lib/gobstones/position.rb

```diff
@@ -0,0 +1,20 @@
+class Position
+  attr_reader :x, :y
+
+  def initialize(x, y)
+    @x = x
+    @y = y
+  end
+
+  def ==(other)
+    self.class === other and
+        other.x == @x and
```

Comment:
[and is the same as && but with lower precedence. They both use short-circuit evaluation.](http://stackoverflow.com/a/1426835/1188303)